lib/rust/ensogl/example/slider/Cargo.toml

```diff
@@ -13,4 +13,5 @@ ensogl-core = { path = "../../core" }
 ensogl-hardcoded-theme = { path = "../../app/theme/hardcoded" }
 ensogl-text-msdf-sys = { path = "../../component/text/msdf-sys" }
 ensogl-selector = { path = "../../component/selector" }
+profiling = { path = "../../../profiling" }
```

Comment:
This is bad:
1. This is an unnecessary change.
2. This makes the dependencies messy again. This is part of EnsoGL core, which is basis for the profiling library, which ... is imported to EnsoGL core. We've been already talking about it during previous PR review. Please browse your code VERY carefully and fix all the things we've been chatting about during the last PR review.